Api.js

```diff
@@ -0,0 +1,112 @@
+export default class ApiClient {
+  constructor() {
+    this.url = 'https://kaozdl-target.herokuapp.com/api/v1/';
+  }
+  //Generic rest client
+  async create(model, data) {
+    try {
+      let response = await fetch(this.url + model + '/', {
+        method: 'POST',
+        headers: {
+          Accept: 'appplication/json',
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(data),
+      },
+      );
+      return response;
+    }
+    catch (error) {
+      return error;
+    }
+  }
+  async retrieve(model, id) {
+    try {
+      let response = await fetch(this.url + model + '/' + id).then(data);
```

Comment:
You can use string substitution on es6.
```suggestion
      let response = await fetch(`${this.url}model/${this.id}`).then(data);
```

Reply:
Thank's for the suggestion, that looks nicer